packages/typescript/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "hops-typescript",
+  "version": "11.0.0-rc.11",
+  "description": "TypeScript support for Hops applications",
+  "keywords": [
+    "hops",
+    "typescript"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=8.10.0"
+  },
+  "files": [
+    "mixin.core.js",
+    "preset.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xing/hops.git"
+  },
+  "dependencies": {
+    "@untool/core": "^0.10.0",
+    "ts-loader": "^4.4.2",
+    "typescript": "^2.9.2"
```

Comment:
I'm currently unsure whether we should add typescript as a dependency or peerDependency.
Having it as a dependency adds convenience that users just need to add the "hops-typescript" plugin and everything works.
On the other hand, having it as a peerDependency means that the user needs to install it themselves - but also allows them to choose a different version if they need it (and it reduces peerDependency warnings when using other packages that expect typescript to be installed - eg. tslint, etc).